### ISSUE TITLE
iOS password manager integration

### ISSUE THREAD

**Issue by jswetzen** [lesspass/lesspass#595]
I would love to switch over to LessPass from LastPass, but the feature I use every day is the native integrations into iOS passwords. It lets me open the password app and select the password so that it’s filled automatically. I still haven’t tried the daily UX of manually switching to the LessPass app, but it’s definitely going to be less convenient.

If this is welcomed, I may well be able to contribute as I’m a software engineer with a Mac.

**Comment by guillaumevincent** [comment]
Sorry for the late answer,
I'm not very active right now because of a little open source fatigue.

To be honest I'm a little newbie on iOS development.
@jswetzen this is very welcome. I will be very happy to integrate this in LessPass, but I'm not sure how. Do you want to try a POC in a pull request ?

**Comment by jswetzen** [comment]
Open source fatigue: I can relate.

In the end I'm afraid I found self-hosted Bitwarden with bitwarden_rs instead and it was just so easy to move over all of my LastPass data there. I really like the idea of a stateless manager, but lazyness often wins.

I found [this talk from WDC](https://developer.apple.com/videos/play/wwdc2018/721/) when looking into iOS password autofill, but I won't be doing a POC at the moment. It should be easy enough though. For iOS users I still think it would be a real nice-to-have, but feel free to close this if you like.
